### ISSUE TITLE
Why fine-tuning always start with 0 accuracy although I'm using the pretrained model as a checkpoint

### ISSUE THREAD

**Issue by MohieEldinMuhammad** [PaddlePaddle/PaddleOCR#9863]
This my config.yml file:

```
Global:
  debug: false
  use_gpu: true
  epoch_num: 300
  log_smooth_window: 20
  print_batch_step: 1
  save_model_dir: ./output/v3_arabic_mobile
  save_epoch_step: 3
  eval_batch_step: 44
  cal_metric_during_train: true
  pretrained_model:
  checkpoints: arabic_PP-OCRv3_rec_train/best_accuracy
  save_inference_dir:
  use_visualdl: false
  infer_img: 
  character_dict_path: ppocr/utils/dict/arabic_dict.txt
  max_text_length: &max_text_length 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3_arabic.txt


Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 0
    epslion: 0.00009
  regularizer:
    name: L2
    factor: 3.0e-05


Architecture:
  model_type: rec
  algorithm: SVTR
  Transform:
  Backbone:
    name: MobileNetV1Enhance
    scale: 0.5
    last_conv_stride: [1, 2]
    last_pool_type: avg
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 64
            depth: 2
            hidden_dims: 120
            use_guide: True
          Head:
            fc_decay: 0.00001
      - SARHead:
          enc_dim: 512
          max_text_length: *max_text_length

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - SARLoss:

PostProcess:  
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: False

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./
    ext_op_transform_idx: 1
    label_file_list:
    - ./trian.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape: [48, 320, 3]
    - RecAug:
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 64
    drop_last: false
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./
    label_file_list:
    - ./test.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 64
    num_workers: 4

```

If the pre-trained model is used why the accuracy starts from zero and then increases shouldn't the accuracy be better from the beginning as any traditional transfer learning?

**Comment by MohieEldinMuhammad** [comment]
@WenmuZhou Please let me know if I'm doing anything wrong that makes the training start from scratch, i want to continue on the existing pre-trained model.

**Comment by ruogedol** [comment]
Maybe it's just that the dictionary at the time of pre-training is different from the dictionary released now. 
When you stop training and choose the best model to migrate again, the dictionary is consistent with the previous training, and the acc will not start from scratch.

**Comment by MohieEldinMuhammad** [comment]
@ruogedol Thanks for replying, Yes you are right i tried using one of my fine-tuned models and it didn't start from scratch. 

This means there is an inconstancy between my config and best_accuracy model config, but i don't think it's in the dictionary as i'm using the same as in the best_accuracy config file!

**Comment by MohieEldinMuhammad** [comment]
Do you have any ideas about what else may make this inconsistency?  @ruogedol  i realy need to continue on the best accuracy model don't want to start from scratch

**Comment by WenmuZhou** [comment]
can you provide logs？

**Comment by MohieEldinMuhammad** [comment]
@WenmuZhou 

I'm very sorry for the late reply, here are my logs 
[train_logs.log](https://github.com/PaddlePaddle/PaddleOCR/files/11444076/train_logs.log)

**Comment by MohieEldinMuhammad** [comment]
@WenmuZhou

**Comment by MohieEldinMuhammad** [comment]
you have to reverse your text in the .txt file as I believe paddle data loader read the data text from left to right

**Comment by MostafaWassel** [comment]
Hello guys, i face the same issue any updates for solving it?

**Comment by Sayaka91** [comment]
@MostafaWassel Not any updates. Although I reserved my dictionary but the accuracy is always 0
